README.md

```diff
@@ -23,9 +23,13 @@ git submodule update --init
 
 To install GUDHI, you can follow the [C++ compilation procedure](https://gudhi.inria.fr/doc/latest/installation.html), the [Python compilation procedure](https://gudhi.inria.fr/python/latest/installation.html), use our [conda-forge package](https://gudhi.inria.fr/conda/), or [go with Docker](https://gudhi.inria.fr/dockerfile/).
 
+# Contribution
```

Comment:
The text below precises it, but if I have a question, I might not think to look in a section called "contributing", that sounds like something for developers who want to add code to gudhi, not something for a "simple" user.